src/odemis/util/peak.py

```diff
@@ -33,34 +33,58 @@
 # warnings.filterwarnings('error')
 # numpy.seterr(all='raise')
 
-# rough estimation of peak width based on fitting type
-PEAK_WIDTHS = {'gaussian': 0.1, 'lorentzian': 1e-4}
+# initial peak width parameters
+PEAK_WIDTHS = {'gaussian_space': 0.1, 'lorentzian_space': 0.1, 'gaussian_energy': 0.1, 'lorentzian_energy': 0.1}
 
 # These two fitting functions are called back from curve_fit()
 # Note: when returning NaN, curve_fit() appears to not like the proposed parameters
 
+
 def GaussianFit(data, *peaks):
     """
     Applies gaussian fitting to data given the "peaks" parameters.
-    peaks (list of floats): series of pos, width, amplitude and initial offset
+    data (1d array of floats): the wavelength list for each N pixel
+    peaks (list of floats): series of pos, width, amplitude and initial offset (given in space domain)
     """
     gau = peaks[-1]  # Automatically converted to a vector in the addition
     for pos, width, amplitude in _Grouped(peaks[:-1], 3):
-        sprime = pos * abs(width)
-        gau += abs(amplitude) * _Normalize(numpy.exp(-(data - pos) ** 2 / sprime ** 2))
+        gau += abs(amplitude) * numpy.exp(-(data - pos) ** 2 / (2 * abs(width) ** 2))
+    return gau
+
 
+def GaussianEnergyFit(data, *peaks):
+    """
+    Applies gaussian fitting to data given the "peaks" parameters.
+    data (1d array of floats): the energy list for each N pixel
+    peaks (list of floats): series of pos, width, amplitude and initial offset (currently given in the energy domain)
+    """
+    gau = peaks[-1] * constant / data**2
```

Comment:
Please add a comment on why the offset is done differently (ie, it's a constant in space domain, so converted to energy here).